Mathlib/Topology/Sheaves/LocalPredicate.lean

```diff
@@ -34,13 +34,10 @@ to the types in the ambient type family.
 We give conditions sufficient to show that this map is injective and/or surjective.
 -/
 
-
-universe v
-
 noncomputable section
 
-variable {X : TopCat.{v}}
-variable (T : X → Type v)
+variable {X : TopCat}
+variable (T : X → Type*)
```

Comment:
I would prefer it the universes were made explicit in the variable declarations throughout.

Reply:
Indeed I sometimes see the universe variable in `TopCat` get inferred by unification and fails to be an independent variable as intended. But `Type*` ensures the universe is independent so it's okay, right? I try to avoid specifying universes unless necessary following [discussions here](https://leanprover.zulipchat.com/#narrow/channel/345428-mathlib-reviewers/topic/universe.20variables/near/501066589).